packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -173,7 +178,7 @@ export function ScheduleAmountCell({
           overflow: 'hidden',
           textOverflow: 'ellipsis',
         }}
-        title={(isApprox ? 'Approximately ' : '') + str}
+        title={isApprox ? t('Approximately {{str}}', { str }) + ' ' : str}
```

Comment:
Same comments as above:
```suggestion
        title={isApprox ? t('Approximately {{currencyAmount}}', { currencyAmount: str }) : str}
```